common/version/semver.go

```diff
@@ -9,7 +9,7 @@ import (
 
 var _ fmt.Stringer = (*Semver)(nil)
 
-// Semver represents a semantic version.
+// Semver represents a semantic defaultVersion.
```

Comment:
Rename mistake, should still be "version"

Reply:
fixed here and elsewhere